content/docs/rfcs/23/README.md

```diff
@@ -0,0 +1,155 @@
+
+The default PubSub topic SHOULD be used for all protocols.
+However, in certain situations other topics MAY be used.
+
+### Pubsub topic format
+
+PubSub topics SHOULD follow the folloing structure:
+
+`/waku/2/TopicName/Encoding`
+
+This namespaced structure makes things like compatibility and discoverability and automatic handling of new topics easier.
+For example, if the encoding of the payload is changed, compression is introduced, etc.
+
+For more on this format of PubSub topics, see [Ethereum 2 P2P spec](https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/p2p-interface.md#topics-and-messages) where inspiration for this format was taken from.
+
+### Separation of two applications example
+
+Let's say we have some M2M topic that is heavy and experience a lot of heavy traffic and a Status Network one. This can be segregated into:
+
+```
+/waku/2/waku-status/proto
+/waku/2/waku-walletconnect/proto
+```
+
+This indicates that they are WakuMessages but for different domains completely.
+
+### Topic sharding example
+
+Topic sharding is currently not supported by default, but is planned for the future in order to deal with increased network traffic. Here's a sketch for what this might look like:
+
+```
+waku/2/waku-9_shard-0/proto
+...
+waku/2/waku-9_shard-9/proto
+```
+
+This indicates explicitly that the network traffic has been partitioned into 10 buckets.
+
+### Compression example
+
+Not yet implemented, but would be easy to add with:
+
+`/waku/2/default-waku/proto_snappy`
+
+### Why use a single PubSub topic by default?
+
+Using a single PubSub topic ensures a connected network, as well some degree of metadata protection. See [section on Anonymity/Unlinkability](/spec/10/#anonymity--unlinkability).
+
+If you use another PubSub topic, be aware that metadata protection might be weakened,
+and other nodes in the network might not subscribe or store messages for your given Pubsub topic.
+This means you are likely to have to run your own full nodes which may make your application less robust.
+
+## Content topics
+
+The other type of topic that exists in Waku v2 is content topics.
+This is used for content based filtering.
+See [14/WAKU2-MESSAGE spec](/spec/14) for where this is specified.
+Note that this doesn't impact routing of messages between relaying nodes,
+but it does impact how request/reply protocols such as 
+[12/WAKU2-FILTER](https://rfc.vac.dev/spec/12/) and [13/WAKU2-STORE](https://rfc.vac.dev/spec/13/) are used.
+
+This is especially useful for nodes that have limited bandwidth,
+and only want to pull down messages that match this filter.
+
+Since all messages are relayed regardless of content topic, you MAY use any content topic you wish without impacting how messages are relayed.
+
+### Content topic format
+
+The format of content topics is as follows:
+
+`/waku/2/ContentTopic/Encoding`
+
+The name of ContentTopic is application-specific. As an example, here's the content topic used for an upcoming testnet:
```

Comment:
```suggestion
The name of ContentTopic is application-specific.
As an example, here's the content topic used for an upcoming testnet:
```

Back to https://github.com/vacp2p/rfc/pull/383/files#r640300187, maybe it is worth naming the field. Are you saying that the second to last field is called "name"?